### DISCUSSION TITLE
whisper permission denied?

### DISCUSSION THREAD

**Discussion by yeetus1992** [openai/whisper#1011] [Q&A] [answered]
Hi!

when i go into my whisper python site packages via finder and try to change something in a file and saving that:

`Do you want to save /Library/Python/3.9/site-packages/whisper/decoding.py before closing?`

it gives me:
`
[Errno 13] Permission denied: '/Library/Python/3.9/site-packages/whisper/decoding.py/' How do I fix this? 
`
Would be beyond grateful for help!

Sincerely:)

**Comment by jongwook** [comment, marked as answer]
Files under `/Library/` are typically only editable with the system administrator privilege (like when you run `sudo` commands or authenticate with Touch ID).

If you installed Whisper using `pip install`, using `pip install --user` instead will make it install under your home directory path instead of `/Library`, and it should be easier to edit them.

**Reply by yeetus1992** [reply]
So I tried to do that but that means I have to uninstall the other version of whisper. Yet when `pip3 uninstall whisper` and then I run whisper it's still there..? or do I need to do `pip3 uninstall git+https://github.com/openai/whisper.git`?
because that then gives me `ERROR: You must give at least one requirement to uninstall (see "pip help uninstall")`
